src/transport/Session.h

```diff
@@ -264,6 +272,11 @@ class Session
 
 private:
     FabricIndex mFabricIndex = kUndefinedFabricIndex;
+#if INET_CONFIG_ENABLE_TCP_ENDPOINT
+    // The underlying TCP connection object over which the session is
+    // established.
```

Comment:
Should be clear that this will be null for sessions not over TCP, yes?  At least in API docs for the public bits.

Reply:
Will enhance the comment.